dask/array/core.py

```diff
@@ -1502,10 +1506,6 @@ def name(self, val):
             "please set ``._name``"
         )
 
-    def __iter__(self):
-        for i in range(len(self)):
-            yield self[i]
-
```

Comment:
This change seems unrelated.

Reply:
hmm I have no recollection of deleting that code... It should re-appear in my next commit

Reply:
yeah probably just a weird merge thing.